src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Manufacturer get(Long id);
```

Comment:
Consider changing the return type of the get(Long id) method to Optional<Manufacturer> instead of Manufacturer. This is a best practice when an object may not exist, thereby avoiding returning null and handling potential NullPointerExceptions.